main.py

```diff
@@ -1,7 +1,33 @@
 import init_django_orm  # noqa: F401
 
 from django.db.models import QuerySet
+from db.models import Genre, Actor
 
 
 def main() -> QuerySet:
     pass
+    genres = [
+        Genre(name="Western"),
+        Genre(name="Action"),
+        Genre(name="Dramma"),
+    ]
+    for genre in genres:
+        genre.save()
```

Comment:
Good practive will be :
1. Create dataset with only names of future genres 
2. In for loop call `create()` method with needed name